go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2130,26 +2130,27 @@ func (e *Executor) scheduleNextMigration(ctx context.Context) error {
 		postponeLaunch := row.AsBool("postpone_launch", false)
 		postponeCompletion := row.AsBool("postpone_completion", false)
 		readyToComplete := row.AsBool("ready_to_complete", false)
-		ddlAction := row["ddl_action"].ToString()
+		isImmediateOperation := row.AsBool("is_immediate_operation", false)
 
 		if postponeLaunch {
 			// We don't even look into this migration until its postpone_launch flag is cleared
 			continue
 		}
 
 		if !readyToComplete {
-			// Whether postponsed or not, CREATE and DROP operations are inherently "ready to complete"
-			// because their operation is instantaneous.
-			switch ddlAction {
-			case sqlparser.CreateStr, sqlparser.DropStr:
+			// see if we need to update ready_to_complete
+			if isImmediateOperation {
+				// Whether postponsed or not, CREATE and DROP operations, as well as VIEW operations,
```

Comment:
```suggestion
				// Whether postponed or not, CREATE and DROP operations, as well as VIEW operations,
```

Reply:
(not sure if this typo exists in the original bugfix, but it caught my eye so mentioning it in any case)